FIAT/__init__.py

```diff
@@ -36,49 +37,52 @@
 
 __version__ = pkg_resources.get_distribution("fenics-fiat").version
 
 # List of supported elements and mapping to element classes
-supported_elements = {"Argyris": Argyris,
-                      "Bell": Bell,
-                      "Bernstein": Bernstein,
-                      "Brezzi-Douglas-Marini": BrezziDouglasMarini,
-                      "Brezzi-Douglas-Fortin-Marini": BrezziDouglasFortinMarini,
-                      "Bubble": Bubble,
-                      "FacetBubble": FacetBubble,
-                      "Crouzeix-Raviart": CrouzeixRaviart,
-                      "Discontinuous Lagrange": DiscontinuousLagrange,
-                      "S": Serendipity,
-                      "DPC": DPC,
-                      "Discontinuous Taylor": DiscontinuousTaylor,
-                      "Discontinuous Raviart-Thomas": DiscontinuousRaviartThomas,
-                      "Hermite": CubicHermite,
-                      "Lagrange": Lagrange,
-                      "Gauss-Lobatto-Legendre": GaussLobattoLegendre,
-                      "Gauss-Legendre": GaussLegendre,
-                      "Gauss-Radau": GaussRadau,
-                      "Morley": Morley,
-                      "Nedelec 1st kind H(curl)": Nedelec,
-                      "Nedelec 2nd kind H(curl)": NedelecSecondKind,
-                      "Raviart-Thomas": RaviartThomas,
-                      "Regge": Regge,
-                      "EnrichedElement": EnrichedElement,
-                      "NodalEnrichedElement": NodalEnrichedElement,
-                      "TensorProductElement": TensorProductElement,
-                      "BrokenElement": DiscontinuousElement,
-                      "HDiv Trace": HDivTrace,
-                      "Hellan-Herrmann-Johnson": HellanHerrmannJohnson}
+supported_elements = {
+    "Argyris": Argyris,
+    "Bell": Bell,
+    "Bernstein": Bernstein,
+    "Brezzi-Douglas-Marini": BrezziDouglasMarini,
+    "Brezzi-Douglas-Fortin-Marini": BrezziDouglasFortinMarini,
+    "Bubble": Bubble,
+    "FacetBubble": FacetBubble,
+    "Crouzeix-Raviart": CrouzeixRaviart,
+    "Discontinuous Lagrange": DiscontinuousLagrange,
+    "S": Serendipity,
+    "DPC": DPC,
+    "Discontinuous Taylor": DiscontinuousTaylor,
+    "Discontinuous Raviart-Thomas": DiscontinuousRaviartThomas,
+    "Hermite": CubicHermite,
+    "Lagrange": Lagrange,
+    "Kong-Mulder-Veldhuizen": KongMulderVeldhuizen,
+    "Gauss-Lobatto-Legendre": GaussLobattoLegendre,
+    "Gauss-Legendre": GaussLegendre,
+    "Gauss-Radau": GaussRadau,
+    "Morley": Morley,
+    "Nedelec 1st kind H(curl)": Nedelec,
+    "Nedelec 2nd kind H(curl)": NedelecSecondKind,
+    "Raviart-Thomas": RaviartThomas,
+    "Regge": Regge,
+    "EnrichedElement": EnrichedElement,
+    "NodalEnrichedElement": NodalEnrichedElement,
+    "TensorProductElement": TensorProductElement,
+    "BrokenElement": DiscontinuousElement,
+    "HDiv Trace": HDivTrace,
+    "Hellan-Herrmann-Johnson": HellanHerrmannJohnson,
+}
```

Comment:
While this new style is (arguably) better, due to the prevalence of primitive diff and merge tools, this style fix is better done in a separate PR than combined with adding features.  For example, when merging this with changes already made in upstream, typical merge tools will give up because of the lack of granularity.  Although you can resolve the merge manually then, if you happen to make a mistake in merge resolution, a reviewer (or yourself) will not easily spot that, because all GitHub shows is that instead of this list we have that list, but it's not immediately obvious what has changed.

Reply:
I understand. Yes, I can just apply selectively.

Reply:
I re-commited __init__.py and quadrature_schemes.py to only format my new additions.

Reply:
Great, thanks.